src/Core/WinSWCore/Configuration/YamlConfiguration.cs

```diff
@@ -277,12 +284,21 @@ public override string ErrFilePattern
             {
                 get
                 {
-                    if (this.ZipOlderThanNumDaysYamlLog != null)
+                    int? zipolderthannumdays = null;
+
+                    if (!string.IsNullOrEmpty(this.ZipOlderThanNumDaysYamlLog))
                     {
-                        return this.ZipOlderThanNumDaysYamlLog;
+                        if (!int.TryParse(this.ZipOlderThanNumDaysYamlLog, out int zipolderthannumdaysValue))
+                        {
+#pragma warning disable S2372 // Exceptions should not be thrown from property getters
```

Comment:
Without the analyzer installed in this repo these directives should not be committed.

Also, please remove those which has been merged into the code base.

Reply:
Make sense. But when remove all #pragma warnings, build is failing. How to resolve this?

Reply:
Tweak your own build environment.

Reply:
This is unrelated to Visual Studio or .NET SDK.